src/top/dashboard.rs

```diff
@@ -121,6 +119,13 @@ impl<'a> Widgets<'a> {
     /// Renders a components table, showing sources, transforms and sinks in tabular form, with
     /// statistics pulled from `ComponentsState`,
     fn components_table<B: Backend>(&self, f: &mut Frame<B>, state: &state::State, area: Rect) {
+        // Header columns
+        let header = ["Name", "Kind", "Type", "Events", "Bytes", "Errors"]
```

Comment:
Not a big deal, but it seems like we could have left the constant and still did the mapping in here.

Reply:
Ah, that's a remnant from my experiments with `pad`, where column labels could occupy different widths depending on padding. This can indeed be static.